requirements.txt

```diff
@@ -1 +1 @@
-numpy
+numpy-financial
```

Comment:
Could you pin the current version of numpy-financial and add numpy pinned version too?

numpy-financial is compatible with:
 - [numpy>=1.15](https://github.com/numpy/numpy-financial/blob/e3c2a4eccd1f44082597ace0550eb3daf20156fd/setup.py#L58)

But new versions they are testing a new feature but currently is consuming a lot of memory or processor.

I think a safe pinned is better in this case where it will work fine without the issues related:
 - https://github.com/Vauxoo/maintainer-quality-tools/pull/315

I mean,

```txt
numpy==1.15
numpy-financial== 1.0.0
```

Reply:
Done